src/css/structure/modules/infragistics.ui.editors.css

```diff
@@ -226,3 +226,8 @@
 .ui-igedit-textarea {
 	resize: none;
 }
+
```

Comment:
You can move that into src\css\themes\bootstrap4\default\modules\_infragistics.ui.editors.scss file, because it is specifically for bootstrap4

Reply:
The suggested changes have been made to all three versions.